code/muilty_rtspsender_gui/rtsp_server.py

```
@@ -0,0 +1,1830 @@
+- 실시간 상태 모니터링
+- MediaMTX 기반 RTSP 송출
+
+요구사항:
+- FFmpeg 설치 필요
+- MediaMTX 설치 권장
+- python-opencv (cv2)
+- tkinter (GUI)
+"""
+
+import os
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)